doc/reference/configuration/configuration_reference.rst

```diff
@@ -1680,6 +1680,135 @@ The ``groups`` section provides the ability to define the :ref:`full topology of
     For example, :ref:`iproto <configuration_reference_iproto>` and :ref:`database <configuration_reference_database>` configuration parameters defined at the group level are applied to all instances in this group.
 
 
+..  _configuration_reference_memtx:
+
+memtx
```

Comment:
The entire `memtx` section should be after `instances`. Now it beaks the `groups -> replicasets -> instances hierarchy`. 

<img width="360" alt="image" src="https://github.com/tarantool/doc/assets/38073144/216a9411-9cc4-4cdb-8da1-385f354e4f2e">

Reply:
Thanks, fixed!